maturity-model.md

```diff
@@ -0,0 +1,175 @@
+* CB-2:	Members of the organization and teams actively seek opportunities to collaborate. Teams routinely discuss, revisit and debate the outcomes of their collaborative efforts, and make these outcomes available by default.
+* CB-3:	Members of the organization collaborate both internally and externally in ways that benefit all involved. Teams routinely discuss, revisit and debate the outcomes of their collaborative efforts, and share their learnings outside the organization, and make these outcomes externally available by default.
+
+### Community
+
+**Sharing Policies**
+
+* SP-0: No sharing culture nor written policies.
+* SP-1:	Some members of the organization unite to define values and principles, but are not clearly supported when they do.
+* SP-2:	Members of the organization collectively document shared visions and agreements like mission statements and codes of conduct, make them easily accessible, and reference them often. Onboarding materials and orientation rituals provide adequate context for helping new members understand how the organization will benefit from their contributions.
+* SP-3:	Shared values and principles inform decision-making, conflict resolution, and assessment processes among members of the organization, who reference these values and principles consistently in both verbal and written formats.
+
+**Feel part of the Organization**
+
+* PA-0:	Low engagement, no collaboration and people do not feel comfortable sharing with others.
+* PA-1:	Members of the organization feel comfortable sharing their thoughts and opinions without fear of retribution, but only in familiar domains. People understand that the best ideas win, and leadership responsibilities accrue to people with histories of contribution and commitment.
+* PA-2:	Members of the organization feel comfortable sharing their thoughts and opinions without fear of retribution. Leaders demonstrate dedication to the organization's shared values.
+* PA-3:	The organization is proactive in telling members that it benefits from their contributions; as such, members demonstrate shared consciousness and empowered execution, and feel a sense of responsibility to the community. Leaders understand that they grow by helping others grow, and they mentor junior members of the organization.
+
+### Governance
+
+**Rewards**
+
+* RW-0:	No rewards.
+* RW-1:	Leaders are encouraged to reward exceptional collaborations, but there are no policies or processes established.
+* RW-2:	Standard processes are established to reward collaborations outside the developers' teams. Team leaders or boards decide who has to be rewarded.
+* RW-3:	Rewards are not only proposed by the organization, but the communities are able to define their more valuable rewards. The community is responsible to decide who has to be rewarded.
+
+**Monitoring Policies**
+
```

Comment:
While important, this needs to be formulated in a much more InnerSource specific way to make sense in that context. Also it could be interpreted two ways:

InnerSource projects using metrics internally to track deployment performance, to track project progress.

InnerSource program metrics to track InnerSource adoption.

All of these are important, but I believe we should make explicit which we are talking about here.

Reply:
I'd say this should include both cases, projects self-assessing and to monitor the InnerSource initiative as a whole.

What about the following?

MP-0: No existing monitoring policies at any level in the organization.
MP-1:  Metrics are important for certain teams, and they start using them in an isolated way.
MP-2: There is a strategy at the organizational level with respect to metrics that help to validate specific policies across the organization. This monitoring policy exists at the level of some of the InnerSource projects.

Reply:
+1, I've changed the wording as per your suggestion and added some additional rationale to the section.